tsconfig.json

```diff
@@ -2,19 +2,15 @@
   "include": ["lib/**/*.js", "test/**/*.jsx", "test/**/*.js", "index.js"],
   "exclude": ["**/*.min.js"],
   "compilerOptions": {
-    "target": "ES2020",
-    "lib": ["ES2020", "DOM"],
-    "module": "ES2020",
-    "moduleResolution": "node",
+    "target": "es2021",
+    "lib": ["es2020", "dom"],
+    "module": "node16",
     "jsx": "react",
     "allowJs": true,
```

Comment:
Same nit as before:

```suggestion
```

Looks great otherwise! :+1:

Reply:
Thanks for the reminder!
Updated